Description: Use unicode-data instead of downloading external files
 Replace external links with copies of Unicode data and GNU Unifont
 provided by the packages unicode-data and ttf-unifont, respectively.
Author: Peter Colberg <peter@colberg.org>
Forwarded: not-needed
Last-Update: 2017-07-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/data/Makefile
+++ b/data/Makefile
@@ -20,13 +20,13 @@
 	$(RUBY) data_generator.rb < UnicodeData.txt > $@
 
 # GNU Unifont version for font metric calculations:
-UNIFONT_VERSION=9.0.04
+UNIFONT_VERSION=10.0.02
 
 unifont.ttf:
-	$(CURL) $(CURLFLAGS) -o $@ $(URLCACHE)https://mirrors.kernel.org/gnu/unifont/unifont-$(UNIFONT_VERSION)/unifont-$(UNIFONT_VERSION).ttf
+	cp /usr/share/fonts/truetype/unifont/unifont.ttf $@
 
 unifont_upper.ttf:
-	$(CURL) $(CURLFLAGS) -o $@ $(URLCACHE)https://mirrors.kernel.org/gnu/unifont/unifont-$(UNIFONT_VERSION)/unifont_upper-$(UNIFONT_VERSION).ttf
+	cp /usr/share/fonts/truetype/unifont/unifont_upper.ttf $@
 
 %.sfd: %.ttf
 	$(FONTFORGE) -lang=ff -c "Open(\"$<\");Save(\"$@\");Quit(0);"
@@ -35,31 +35,31 @@
 	$(JULIA) charwidths.jl > $@
 
 # Unicode data version
-UNICODE_VERSION=9.0.0
+UNICODE_VERSION=10.0.0
 
 UnicodeData.txt:
-	$(CURL) $(CURLFLAGS) -o $@ -O http://www.unicode.org/Public/$(UNICODE_VERSION)/ucd/UnicodeData.txt
+	cp /usr/share/unicode/UnicodeData.txt $@
 
 EastAsianWidth.txt:
-	$(CURL) $(CURLFLAGS) -o $@ -O $(URLCACHE)http://www.unicode.org/Public/$(UNICODE_VERSION)/ucd/EastAsianWidth.txt
+	cp /usr/share/unicode/extracted/DerivedEastAsianWidth.txt $@
 
 GraphemeBreakProperty.txt:
-	$(CURL) $(CURLFLAGS) -o $@ -O $(URLCACHE)http://www.unicode.org/Public/$(UNICODE_VERSION)/ucd/auxiliary/GraphemeBreakProperty.txt
+	cp /usr/share/unicode/auxiliary/GraphemeBreakProperty.txt $@
 
 DerivedCoreProperties.txt:
-	$(CURL) $(CURLFLAGS) -o $@ -O $(URLCACHE)http://www.unicode.org/Public/$(UNICODE_VERSION)/ucd/DerivedCoreProperties.txt
+	cp /usr/share/unicode/DerivedCoreProperties.txt $@
 
 CompositionExclusions.txt:
-	$(CURL) $(CURLFLAGS) -o $@ -O $(URLCACHE)http://www.unicode.org/Public/$(UNICODE_VERSION)/ucd/CompositionExclusions.txt
+	cp /usr/share/unicode/CompositionExclusions.txt $@
 
 CaseFolding.txt:
-	$(CURL) $(CURLFLAGS) -o $@ -O $(URLCACHE)http://www.unicode.org/Public/$(UNICODE_VERSION)/ucd/CaseFolding.txt
+	cp /usr/share/unicode/CaseFolding.txt $@
 
 NormalizationTest.txt:
-	$(CURL) $(CURLFLAGS) -o $@ -O $(URLCACHE)http://www.unicode.org/Public/$(UNICODE_VERSION)/ucd/NormalizationTest.txt
+	bzip2 -d < /usr/share/unicode/NormalizationTest.txt.bz2 > $@
 
 GraphemeBreakTest.txt:
-	$(CURL) $(CURLFLAGS) $(URLCACHE)http://www.unicode.org/Public/$(UNICODE_VERSION)/ucd/auxiliary/GraphemeBreakTest.txt | $(PERL) -pe 's,÷,/,g;s,×,+,g' > $@
+	$(PERL) -pe 's,÷,/,g;s,×,+,g' < /usr/share/unicode/auxiliary/GraphemeBreakTest.txt > $@
 
 clean:
 	rm -f UnicodeData.txt EastAsianWidth.txt GraphemeBreakProperty.txt DerivedCoreProperties.txt CompositionExclusions.txt CaseFolding.txt NormalizationTest.txt GraphemeBreakTest.txt CharWidths.txt unifont*.ttf unifont*.sfd
